crates/sui-types/src/effects/effects_v1.rs

```diff
@@ -129,24 +133,113 @@ impl TransactionEffectsAPI for TransactionEffectsV1 {
             })
             .collect()
     }
+
     fn created(&self) -> Vec<(ObjectRef, Owner)> {
         self.created.clone()
     }
+
     fn mutated(&self) -> Vec<(ObjectRef, Owner)> {
         self.mutated.clone()
     }
+
     fn unwrapped(&self) -> Vec<(ObjectRef, Owner)> {
         self.unwrapped.clone()
     }
+
     fn deleted(&self) -> Vec<ObjectRef> {
         self.deleted.clone()
     }
+
     fn unwrapped_then_deleted(&self) -> Vec<ObjectRef> {
         self.unwrapped_then_deleted.clone()
     }
+
     fn wrapped(&self) -> Vec<ObjectRef> {
         self.wrapped.clone()
     }
+
+    fn object_changes(&self) -> Vec<ObjectChange> {
```

Comment:
I would ideally like to write some unit tests for this (and the equivalent for Effects V2). @lxfind, is there a good home for those kinds of tests (run with V1, then with V2, then confirm some aspect of the effects is the same?).